process-migration-service/frontend/README.md

```diff
@@ -0,0 +1,33 @@
+# React GUI for Process Instance Migration application 
+
+## PIM React GUI can run in two modes
+
+### Full function mode
+This mode needs PIM-GUi and PIM-Service build together in parent folder though maven. 
+
+Please note Thorntail configuration file needs to have "kieServersIDs" defined. For more details see PIM-Service/README.md
```

Comment:
This is no longer needed as now we obtain KIE Server IDs directly form KIE Servers themselves. We just need URL and credentials to connect.